<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana de Inicio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_ingresa_nombre">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresa el nombre de tu bailar√≠n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombre_usuario">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_ver_ranking">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver Ranking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_comenzar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Comenzar</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo_vinicio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
